functionary/ui/templates/core/workflow_list.html

```diff
@@ -1,24 +1,39 @@
 {% extends "base.html" %}
 {% block content %}
-    <div class="d-flex justify-content-between">
+    <div class="d-flex justify-content-between align-items-center">
         <nav aria-label="breadcrumb">
-            <ol class="breadcrumb">
+            <ol class="breadcrumb mb-0">
                 <li class="breadcrumb-item active" aria-current="page">Workflow List</li>
             </ol>
         </nav>
-        {% if request.session.user_can_create_workflow %}
-            <a class="btn btn-primary fa-plus"
-               type="button"
-               title="Create New Workflow"
-               hx-target="body"
-               hx-swap="beforeend"
-               hx-get="{% url 'ui:workflow-create' %}">
-                Create Workflow
-            </a>
+        <div>
+            {% if request.session.user_can_create_workflow %}
+                <a class="btn btn-primary btn-sm fa-plus py-1 h-75"
+                   type="button"
+                   title="Create New Workflow"
+                   hx-target="body"
+                   hx-swap="beforeend"
+                   hx-get="{% url 'ui:workflow-create' %}">
+                    Create Workflow
+                </a>
+            {% endif %}
+            {% if filter %}
+                <button class="btn btn-primary btn-sm d-xl-none py-1 h-75"
```

Comment:
If this button is going to be the same everywhere, it might be worth turning it into its own partial and including it.  That way if it needs to be tweaked it only has to happen in one place.